layouts/shortcodes/observability_pipelines/configure_existing_pipelines/install_worker/linux_apt.md

```diff
@@ -0,0 +1,29 @@
+
+1. Click **Select API key** to choose the Datadog API key you want to use.
+1. Run the one-step command provided in the UI to re-install the Worker:
+    ```shell
+    DD_API_KEY=<datadog_api_key> DD_OP_PIPELINE_ID=<pipeline_id> DD_SITE=<datadog_site> <source_env_variables> <destination_env_variables> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_op_worker2.sh)"
```

Comment:
```suggestion
    DD_API_KEY=<datadog_api_key> DD_OP_PIPELINE_ID=<pipeline_id> DD_SITE=<datadog_site> <source_env_variables> <destination_env_variables> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_op_worker2.sh)"
```
Capitalize placeholders like datadog_api_key? I can't tell if these are autopopulated here. Same query for the commands below on this same page.

Reply:
Capitalized the placeholders in all sections.